utils_linux.go

```diff
@@ -272,6 +273,13 @@ func (r *runner) run(config *specs.Process) (int, error) {
 	if err = r.checkTerminal(config); err != nil {
 		return -1, err
 	}
+	// configure linux personality
+	if r.container.Config().Personality != nil {
+		err = setLinuxPersonality(r.container.Config().Personality.Domain)
```

Comment:
Should we `fail` for this err ?  I low-key want to ignore this err if `syscall.SYS_PERSONALITY` fails  proceed as if nothing happened. I'll probably wait for mods to suggest here.

Reply:
Failing seems better for security.

I guess we should also fail when unknown `Personality.Flags` are set

Reply:
@AkihiroSuda Cool i agree it makes sense from security perspective so i will let the check be there. For `Personality.Flags` i did not implement it as `runtime-spec` suggests that no flag values are supported as well as domain is limited to `LINUX` and `LINUX32`. So anything unknown defaults to `LINUX` i.e `0x0000`.